content/en/logs/processing/_index.md

```diff
@@ -117,7 +117,7 @@ Using the Datadog Agent or the RFC5424 format automatically sets the host value
 
 ### *source* attribute
```

Comment:
Add a forwarding alias to the front matter for the deleted guide. On line 5 insert:
```
aliases:
  - /logs/faq/integration-pipeline-reference
```

Reply:
done